Makefile

```diff
@@ -20,7 +20,10 @@ clean-docs:
 	rm -rf docs/user_guide/jupyter_execute
 
 lint:
-	$(BIN)python -m pylint pylabrobot
+	$(BIN)python -m ruff check pylabrobot && $(BIN)python -m pylint pylabrobot
```

Comment:
do we need both?

Reply:
i'd love to get rid of pylint, but it seems like you did a lot of work on it. 

are you opposed to me just rm'ing it?

Reply:
some differences are that ruff seems less strict about line len. makes pylint fail

Reply:
it's mostly google's template, and i added some small things. as long as behavior is roughly the same it should be good to rm here

Reply:
wdym less strict about line len? why not always <100?